src/templates/work-day-information.jsx

```
@@ -8,6 +8,7 @@ import { renderRichText } from "gatsby-source-contentful/rich-text";
 import { parseMeetingPoint } from "../helpers/parse-meeting-point";
 import * as style from "./work-day-information.module.css";
 import { contentfulRenderingOptions } from "../helpers/contentful-rendering-options";
+import Map from "../components/map/map";
```

Biome (2.4.6)

[error] Do not shadow the global "Map" property.

(lint/suspicious/noShadowRestrictedNames)